src/test/unit/algorithms/ConnectionsTest.cpp

```diff
@@ -29,6 +29,7 @@ namespace testing {
 using namespace std;
 using namespace htm;
 
+// EPSILON used for testing, Connections uses nupic::Epsilon internally
 #define EPSILON 0.0000001
```

Comment:
OT but why is this not the same? Could you replace, or at least set to the same value?